cdisc_rules_engine/operations/label_referenced_variable_metadata.py

```diff
@@ -0,0 +1,23 @@
+from cdisc_rules_engine.operations.base_operation import BaseOperation
+import pandas as pd
+
+
+class LabelReferencedVariableMetadata(BaseOperation):
```

Comment:
Wouldn't it be better to have a single Operation where we would pass in "name" or "label" as an additional parameter?

Reply:
We could, but I figured it would be easier on the rule authors to know there are two reference operations than to remember a specific parameter for this operation. I also didn't think that parameter would be used elsewhere so I didn't think we should extend the schema